### PR TITLE
[Benchmark] Add `export_batch` and `export_batch_if` benchmark

### PR DESCRIPTION
**Pull request by rhdong**
- Fix a little potential bug for exporting RECLAIM KEYS
- Some name refactoring

### CODE REVIEW

**Review by yao-jiashu — APPROVED** [review]

File: README.md [hunk 1]
```diff
@@ -115,9 +115,7 @@ Your environment must meet the following requirements:
 - CUDA version >= 11.2
 - NVIDIA GPU with compute capability 8.0, 8.6, 8.7 or 9.0
 
-
-## Benchmark & Performance(W.I.P)
-
+## Benchmark
```
Comment (jiashuy) [inline README.md:118]: Why remove "Work in Progress"? I think there are some goals to achieve in the future on the Performance.

Comment (rhdong) [reply README.md:118]: You're right, and I mistake in removing it. Let me add back.

**Review by jiashuy — DISMISSED** [review]

**Pull request merged**